src/pages/idea-market/IdeaRegisteredPage.tsx

```diff
@@ -5,15 +5,57 @@ import QnASection from '../../components/postdetail/QnASection';
 import AuthorInfo from '../../components/postdetail/AuthorInfo';
 import styles from './ideaRegisteredPage.module.scss';
 
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
+import { getIdeaMarketDetail } from '../../apis/detailPageAPI';
+import { IdeaMarketDetail } from '../../types/detailPageType';
+
 export const IdeaRegisteredPage = () => {
+  const { ideaId } = useParams<{ ideaId: string }>();
+  //const validIdeaId = ideaId ? Number(ideaId) : undefined;
+
+  const { data, isLoading, error } = useQuery<IdeaMarketDetail, Error>({
+    queryKey: ['ideaMarketDetail', ideaId],
+    queryFn: () => getIdeaMarketDetail(Number(ideaId)),
+    enabled: !!ideaId,
+    staleTime: 1000 * 60 * 5, //5분
+  });
+
+  if (isLoading) return <div>로딩 중...</div>;
+  if (error) return <div>오류 발생!</div>;
+
   return (
     <>
       <div className={styles.margin}>
-        <ProfileHeader />
-        <PostTitlePay />
-        <IdeaDescription />
+        <ProfileHeader
+          name={data?.writer?.name || ''}
+          profileImageUrl={data?.writer?.profileImageUrl || ''}
+          openMyProfile={() => console.log('프로필 페이지로 이동')} // 임시 지정
+        />
+        <PostTitlePay
+          thumbnailImageUrl={data?.thumbnailImageUrl || ''}
+          category={data?.category || ''}
+          ideaMarketType={data?.ideaMarketType || ''}
+          auth={data?.auth || ''}
+          title={data?.title || ''}
+          price={data?.price || 0}
+          viewCount={data?.viewCount || 0}
+          saveCount={data?.saveCount || 0}
+          createdDate={data?.createdDate || ''}
```

Comment:
prop을 하나하나 받지 말고 하나의 객체로 받으면 좋을 것 같아요!

```suggestion
          postData : {
              thumbnailImageUrl={data?.thumbnailImageUrl || ''}
              category={data?.category || ''}
              ideaMarketType={data?.ideaMarketType || ''}
              auth={data?.auth || ''}
              title={data?.title || ''}
              price={data?.price || 0}
              viewCount={data?.viewCount || 0}
              saveCount={data?.saveCount || 0}
              createdDate={data?.createdDate || ''
         }
```